### PR TITLE
JavaScript Best Practice Rev1

### PR DESCRIPTION
**Pull request by RHK-MICROVERSE**
Request for review by the coding partner

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+<body>
+  <div class="mother-container">
+    <ul class="main-container">
+      <li class="title">
+        <h4>Today's To Do</h4>
+        <i class="fa-solid fa-arrows-rotate"></i>
+      </li>
+      <li class="form-input">
+        <input type="text" id="idiot" class="user-imput" placeholder="Add to your list...">
```
Comment (why-a-en) [inline index.html]: Please consider giving the appropriate name for the id of this input element for better code readability and kindly correct the spelling of "user-input". 🤓

File: index.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        <i class="fa-solid fa-turn-down"></i>
+      </li>
+      <ul id="addtodotasks">
```
Comment (why-a-en) [inline index.html]: I highly suggest naming the id of this ul element with a camel case for better code readability and semantic purposes 🐫 💼

File: index.js [hunk 3]
```diff
@@ -0,0 +1,98 @@
+import addTask, { removeTask } from './modules/addAndRemove.js';
+// import './index.css';
+import DeleteAll from './modules/markCompletion.js';
+// import addTask, { removeTask } from './addAndRemove.js';
+// import DeleteAll from './markCompletion.js';
```
Comment (why-a-en) [inline index.js]: Kindly remove unused comments for clean code and YAGNI 👍

**Review by why-a-en — CHANGES_REQUESTED** [review]
### Hi @RHK-MICROVERSE 

You put a great effort into the project and all the functions work perfectly 👌 💯 
However, I would like to highlight some best practices to improve the quality of your code.

### Highlights
- GitHub flow is correctly used ✅ 
- Descriptive comments used ✅ 

## Status: Changes required ♻️ 

_Please check the comments under the review and make the required changes_




Happy coding! ☕ 👨‍💻

File: index.html (hunk 1 above)
Comment (Rassel-AI) [reply index.html]: id = 'idiot' replaced with id = 'input-user'
class = 'user-imput' replace with class = 'user-input'

File: index.js (hunk 3 above)
Comment (Rassel-AI) [reply index.js]: All unused comment have been removed. Thank you

File: index.html (hunk 2 above)
Comment (Rassel-AI) [reply index.html]: Camel Casing has been applied. Thank you.

**Pull request merged**